README.md

```diff
@@ -1,2 +1,67 @@
-# election-manager
-Manages election configuration, ballot creation, and more.
+# Election Manager
+
+This app is intended to be used on an offline computer by an election admin.
+
+### Features
+- manage election configuration
+- create hand-marked paper ballots
+- export an "election ballot package" for [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager)
+
+## Workflow
+
+1. Admin may create a new election or select an existing election configuration file (`election.json`) to start with (or possibly a previously created election ballot package). 
+2. Admin can edit the election configuration via simple UI forms at any time.
+3. Admin can list all ballots by precinct/style.
+4. Admin can view an individual ballot, both HTML and PDF versions.
+5. Admin can export an election ballot package for [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager).
+
+   Election Ballot Package contains:
+     - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
+     - approved official ballots in pdf format:
+       - one for each combination of ballot style and precinct
+       - file name pattern: `ballot-style-7-precinct-8.pdf`
+
+## App Screens
+- Unconfigured App Screen
+  - Select election config file
+  - Button to create new election
+- Edit Election Config (MVP solution)
+  - Form:
+    - Textarea
+    - Save button
+    - Cancel button (with warning modal, if config has been changed)
+    - Remove config button (with warning modal)
+    - Export config button
+- List Ballot Styles
+  - Rows:
+    - precinct
+    - style
+    - contest count
+    - view ballot link
+  - Filters:
+    - precinct
+    - style
+- View Ballot
+  - html ballot in print-ready format (via PagedJS)
+  - PDF ballot (in iframe under html ballot or click to open in new window)
+  - Previous and Next Ballot Style links
+
+## Navigation
+Displayed when election config exists.
+
+- Ballot Styles
+- Edit Election Config
+- Export Election Ballot Package
+
+## Future Features
+- Election Editor with sections to edit, metadata, parties, contests, ballot styles, etc.
+  - Edit Contest section
+    - List Contests screen
+    - Edit Contest Screen with live preview
+
+## Open Questions
+- Do we implement anything to ensure that ballots are not edited after package is created?
+  - possibly a `ballots.json` with PDF ballot file hashes?
+  - possibly a hash of the package as a separate file?
+
```

Comment:
we could have a manifest of ballot files with hashes, but I think best to start simple with a hash of the election package displayed at creation time and at load time in other components.